src/ILVerification/src/TypeVerifier.cs

```diff
@@ -81,10 +83,17 @@ public void VerifyInterfaces()
                 {
                     VerificationError(VerifierError.InterfaceImplHasDuplicate, type, interfaceType, _module.MetadataReader.GetToken(interfaceHandle));
                 }
-            }
-
-            // Other check
 
+                // Look for missing method implementation
```

Comment:
Skip this part for abstract types. Abstract types can have missing method implementations,

Reply:
Sure

Reply:
done thank's Jan!

Reply:
Add a positive tests for it? (No errors reported with abstract types with missing impls.)

Reply:
Sure I'll add it(I've got also other IL sample tests to add on this, generics, missing parameter, return type, different parameters etc...but I'll add it after merge to avoid too big PR).
However, on this I'll add abstract test!

Reply:
Sounds good.

Reply:
PTAL